mockup1/assets/76f6034.css

```diff
@@ -460,7 +460,7 @@ ul#container > li::before {
 li.accessibility {background-color: #ede5f4}
 li.browser {background-color: #a8c5a9}
 li.css {background-color: #a7cd78}
-li.data {background-color: #94c4b4}
+li.data { position: static; background-color: #94c4b4; }
```

Comment:
Isn't it better to apply that position to all `ul.taglist > li` to avoid the same problem for the other tags?